syntax = "proto3";

package energy_unit;

service Unary {
  rpc GetServerResponse(Message) returns (MessageResponse) {}
}

message Message { string message = 1; }

message MessageResponse {
  string message = 1;
  bool received = 2;
}
